aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -13,11 +13,11 @@ module aptos_std::simple_map {
     const EKEY_ALREADY_EXISTS: u64 = 0;
     const EKEY_NOT_FOUND: u64 = 1;
 
-    struct SimpleMap<Key: store, Value: store> has store {
+    struct SimpleMap<Key: store, Value: store> has drop, store {
```

Comment:
This wouldn't work though right? The Key and Value would need to have drop as well. But that's changing the constraint quite a bit.

Reply:
You need to remove the `:store` (not constraint) and the type parameters. Similar on the `Element` type. This wont compile. See also slack chat.

Reply:
Nvm we discussed offline and this would work 👍

Reply:
@wrwg  the code compiles now. But I can remove the :store on key, value and element to make the simple_map more versatile.